deploy/manager/10_scylla_cluster.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: scylla.scylladb.com/v1
+kind: ScyllaCluster
+metadata:
+  name: scylla-manager-cluster
+  namespace: scylla-manager
+spec:
+  agentVersion: 2.2.0
+  datacenter:
+    name: manager-dc
+    racks:
+    - members: 1
+      name: manager-rack
+      resources:
+        limits:
+          cpu: 1
+          memory: 200Mi
+        requests:
+          cpu: 1
+          memory: 200Mi
+      scyllaAgentConfig: scylla-agent-config
+      scyllaConfig: scylla-config
+      storage:
+        capacity: 5Gi
+  developerMode: true
+  cpuset: true
```

Comment:
is this going to fail in cluster that don't have kubelet set for static cpu policy?

Reply:
It's not going to fail, but performance won't be great.